index.js

```diff
@@ -17,7 +17,11 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 if (app.get('env') === 'development') app.use(morgan('dev'));
 
+<<<<<<< HEAD
```

Comment:
@Musigwa, Did you consider fixing conflicts in this file before raising the PR??

Reply:
Hello @espoirMur, I think there is are no conflicts in this index.js file. Would you please double check the file to make sure? TIA.